modules/api-svc/src/lib/model/InboundTransfersModel.js

```diff
@@ -48,6 +44,9 @@ class InboundTransfersModel {
             bulkTransfersEndpoint: config.bulkTransfersEndpoint,
             transactionRequestsEndpoint: config.transactionRequestsEndpoint,
             bulkQuotesEndpoint: config.bulkQuotesEndpoint,
+            // todo: extend BaseRequestConfigType in sdk-standard-comps
```

Comment:
we have a todo here probably needs to be addressed

Reply:
Removed. Will address in sdk-standard-component PR.